### PR TITLE
deps: Run `yarn upgrade`

### PR DESCRIPTION
**Pull request by chrisbobbe**
This should close all our currently open dependabot PRs: https://github.com/zulip/zulip-mobile/pulls/app%2Fdependabot

### CODE REVIEW

**Comment by chrisbobbe** [comment]
Tested on my iPhone 13 Pro running iOS 16.4, and on the office Android device (Samsung Galaxy S9 running Android 9); all seems fine.

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "expo-sqlite": "~10.2.0",
     "expo-web-browser": "~10.2.1",
     "flow-enums-runtime": "^0.0.6",
+    "hermes-parser": ">=0.4.8",
```
Comment (gnprice) [inline package.json:50]: This happens in the commit whose message is just "deps: Run `yarn upgrade`". I reproduce that `yarn upgrade` causes this line to get added. Odd.

Presumably related to the hermes-parser line in our `resolutions` section. Perhaps drop that? At this point our dependencies that pull it in all depend on newer versions than this anyway.

**Review by gnprice — APPROVED** [review]
Thanks for taking care of this! I guess it's been a little while.

One small comment below.

**Comment by chrisbobbe** [comment]
Thanks for the review! Revision pushed.

**Pull request merged**